pages/help.vue

```diff
@@ -1,7 +1,10 @@
 <template>
-  <div class="bg-white">
+  <div class="">
```

Comment:
The Help page is for people who are having trouble.  We therefore want to keep it simple:

- white background for readability
- no sidebars to allow people to focus on the actual help

This means there is inconsistency with other pages, but deliberately so.